src/components/technologies/genericTechnologies/conv1.py

```diff
@@ -56,9 +63,29 @@ class Conv1(Technology):
       .. math::
          \sum(Input_{t, car}) = 0
 
-    - ``performance_function_type == 3``: Piecewise linear performance function (makes big-m transformation required).
-      The same constraints as for ``performance_function_type == 2`` with the exception that the performance function
-      is defined piecewise for the respective number of pieces
+      If the technology has a standby-power, the input of the standy-by power carrier
+      is:
+
+      .. math::
+         Input_{t, standby-carrier} = standbypower * S
+
+    - ``performance_function_type == 3``: Piecewise linear performance function (
+      makes big-m transformation required). The same constraints as for
+      ``performance_function_type == 2`` with the exception that the performance
+      function is defined piecewise for the respective number of pieces.
+
+    - ``performance_function_type == 4``:Piece-wise linear, minimal partload,
+      includes constraints for slow (>1h) startup and shutdown trajectories.
+      Based on Equations 9-11, 13 and 15 in Morales-España, G., Ramírez-Elizondo, L.,
+      & Hobbs, B. F. (2017). Hidden power system inflexibilities imposed by
+      traditional unit commitment formulations. Applied Energy, 191, 223–238.
+      https://doi.org/10.1016/J.APENERGY.2017.01.089
+
+    - Additionally, ramping rates of the technology can be constraint.
+
+      .. math::
+         -rampingrate \leq \sum(Input_{t, car}) - \sum(Input_{t-1, car})
```

Comment:
didn't check in the code but it should also have \geq RR on the right hand side of the constraint